src/components/AgreementSections/ControllingLicense.js

```diff
@@ -105,6 +125,7 @@ export default class ControllingLicense extends React.Component {
           id={`controlling-license-${statusString}-amendments`}
           license={linkedLicense.remoteId_object}
           renderStatuses={status !== statuses.HISTORICAL}
+          renderNotes={status !== null}
```

Comment:
I _believe_ you're trying to render notes unless the notes are unset. If so, maybe make it more obvious by rewriting this as 

```suggestion
          renderNotes={statusString !== status.UNSET}
```

And while you're at it, change that `renderStatuses` line to check `statusString` instead of `status` as well, for uniformity.

Reply:
It may also just be nicer to go stop relying on this dumb `null -> 'unset'` translation that I do here...

Like, I wonder if you could just go into getLicenseAmendments.js:23 and change the default value from `null` to `statuses.UNSET` (after importing `statuses`, naturally).

Then, change the places where I pass `null` into that function or rely on `null` and instead use `statuses.UNSET`... It'd also let you ditch that `statusString` vs `status` mess in this function...

That'd be ideal I think. If doing that clean up makes you uncertain or you start trying and it messes everything up, it's fine to skip it. I can pick it up afterward.

Reply:
This makes sense, I've changed the default value for an unassigned amendment to be statuses.UNSET, and the code now relies on that, removing the need for statusString at all (I believe)